src/components/Avatar/index.tsx

```diff
@@ -21,31 +25,40 @@ const getInitials = (firstName?: string, lastName?: string) =>
 		lastName?.charAt(0).toUpperCase() || ''
 	}`;
 
-export const Avatar = ({
+const AvatarBase = ({
+	theme,
 	src,
 	firstName,
 	lastName,
 	emphasized,
 	...otherProps
 }: AvatarProps) => {
 	const initials = getInitials(firstName, lastName);
-
 	return (
 		<>
 			{src && (
 				<GrommetAvatar
-					size={emphasized ? '36px' : '24px'}
+					size={emphasized ? px(theme.fontSizes[6]) : px(theme.fontSizes[5])}
 					src={src}
+					background={theme.colors.quartenary.dark}
 					{...otherProps}
 				></GrommetAvatar>
 			)}
 			{initials && !src && (
-				<GrommetAvatar size={emphasized ? '36px' : '24px'} {...otherProps}>
-					<Txt>{initials}</Txt>
+				<GrommetAvatar
+					background={theme.colors.quartenary.dark}
+					size={emphasized ? px(theme.fontSizes[6]) : px(theme.fontSizes[5])}
```

Comment:
NOTE: theme.fontSizes[6] is 34px not 36px

Reply:
Yeah that's fine I think 👍